ocw/lib/gke.py

```diff
@@ -0,0 +1,77 @@
+import os
+import subprocess
+from datetime import datetime, timezone
+import kubernetes
+import googleapiclient.discovery
+from google.oauth2 import service_account
+from ocw.lib.gce import GCE
+
+
+class GKE(GCE):
+    __instances = {}
+
+    def __new__(cls, vault_namespace):
+        if vault_namespace not in GKE.__instances:
+            GKE.__instances[vault_namespace] = self = object.__new__(cls)
+            self.__gke_client = None
+            self.__kubectl_client = {}
+            self.init()
+        return GKE.__instances[vault_namespace]
+
+    def get_project_id(self):
```

Comment:
you keep `__project` field but you introducing this method . it is duplicating of functionality . pick up one or field or function

Reply:
done